apps/backend/core/configuration/base.py

```
@@ -0,0 +1,155 @@
+        pusher_app_id: Pusher app ID
+        pusher_key: Pusher key
+        pusher_secret: Pusher secret
+        pusher_cluster: Pusher cluster
+        clerk_api_key: Clerk API key
+        supabase_url: Supabase URL
+        supabase_key: Supabase key
+        roblox_api_key: Roblox API key
+        sentry_dsn: Sentry DSN
+    """
+
+    def __init__(self):
```

Ruff (0.14.4)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)